src/locales/en/translation.json

```diff
@@ -286,7 +286,10 @@
     "reddit": "Reddit",
     "telegram": "Telegram",
     "twitter": "X",
-    "youtube": "Youtube"
+    "youtube": "Youtube",
+    "linkedin": "LinkedIn",
+    "docs": "Documentation",
+    "home": "Home"
```

Comment:
Isn't it there already definition for `Home` somewhere?

Reply:
No, apparently there isn't.